### PR TITLE
Workout creator

### PR DESCRIPTION
**Pull request by richardpluta**

### CODE REVIEW

File: GrizzFit/src/components/WorkoutsListItem.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    <Text style={styles.buttonMiniText}>
+                        {item.tags}
+                    </Text>
+                </TouchableOpacity>
+            </View>
+            <TouchableOpacity onPress={handleRemove}>
+                <MaterialIcons
+                    name="close"
+                    size={24}
+                    color={darkModePalette.red}
```
Comment (Plootonix) [inline GrizzFit/src/components/WorkoutsListItem.js:50]: sus

Comment (Plootonix) [reply GrizzFit/src/components/WorkoutsListItem.js:50]: i mean it's not that sus i guess. it just makes me nervous thats all.

File: GrizzFit/src/helpers/Helpers.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+
+export const intensityToString = (num) => {
+    const converter = ['Light','Light - Medium','Medium','Medium - Heavy','Heavy']
+    return converter[num * 2]
+}
```
Comment (Plootonix) [inline GrizzFit/src/helpers/Helpers.js:7]: ???

File: GrizzFit/src/navigation/AppStack.js [hunk 3]
```diff
@@ -5,30 +5,37 @@ import Main from '../screens/Main';
...
+              swipeEdgeWidth: 200,
+              headerTitle: () => <Header route={route} />,
+              headerStyle: styles.header,
+              headerLeft: () => <DrawerToggleButton tintColor="#CCCCCC" />,
+              drawerActiveTintColor: "#CC9B6D",
+              drawerInactiveTintColor: "#CCCCCC"
+            })}
+          >
+            <Drawer.Screen name="Main" component={Main} options={{ title: "Home" }} />
+            <Drawer.Screen name="ProfileStack" component={ProfileStack} options={{ title: "Profile" }} />
```
Comment (Plootonix) [inline GrizzFit/src/navigation/AppStack.js:33]: ;)

File: GrizzFit/src/providers/MainScreenProvider.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    return (
+        <MainScreenContext.Provider
+            value={{
+                recordingWorkout,
+                setRecordingWorkout,
+                numberOfCreatedWorkouts,
+                setNumberOfCreatedWorkouts
+            }}
+        >
+            {children}
```
Comment (Plootonix) [inline GrizzFit/src/providers/MainScreenProvider.js:18]: kill it?

File: GrizzFit/src/providers/WorkoutExercisesProvider.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+},
+{
+  key: "3",
+  title: "Built By Science Back Day",
+  tags: "#back #builtbyscience",
+  exercises: [...initialExercises].filter(wrk => wrk.key === "2"),
+  isFavorite: false
+}]
+
+export const WorkoutExercisesProvider = ({children}) => {
```
Comment (Plootonix) [inline GrizzFit/src/providers/WorkoutExercisesProvider.js:63]: kill these

File: GrizzFit/src/screens/Main.js [hunk 6]
```diff
@@ -1,46 +1,67 @@
...
+  const getUserInfo = async () => {
+    const userDoc = await firestore.collection("users").doc(user.uid).get()
+    if (!userDoc.exists) { console.log('No such document!'); }
+    setUserInfo(userDoc.data())
+    setIsLoading(false)
+  }
+
+  useEffect(() => {
+    getUserInfo()
+    navigation.addListener("focus", () => setIsLoading(!isLoading))
```
Comment (Plootonix) [inline GrizzFit/src/screens/Main.js:28]: you're welcome

File: GrizzFit/src/screens/Main.js [hunk 7]
```diff
@@ -1,46 +1,67 @@
...
...Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Et malesuada fames ac turpis egestas maecenas pharetra. Rhoncus mattis rhoncus urna neque viverra justo nec ultrices dui. Amet consectetur adipiscing elit ut aliquam. Massa enim nec dui nunc mattis. Elit ullamcorper dignissim cras tincidunt lobortis feugiat vivamus at augue. Aliquam sem fringilla ut morbi tincidunt. Faucibus et molestie ac feugiat sed lectus vestibulum mattis ullamcorper. Ipsum consequat nisl vel pretium lectus quam. In nisl nisi scelerisque eu ultrices vitae auctor. Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet. Dictum sit amet justo donec enim diam vulputate ut. Mi quis hendrerit dolor magna eget est lorem ipsum. Tincidunt nunc pulvinar sapien et ligula ullamcorper malesuada proin. Nibh mauris cursus mattis molestie a iaculis at erat pellentesque. Amet consectetur adipiscing elit duis tristique sollicitudin nibh. Tempor orci eu lobortis elementum nibh tellus molestie nunc non. Nulla facilisi cras fermentum odio. Fusce ut placerat orci nulla pellentesque. Cras semper auctor neque vitae tempus quam pellentesque nec nam.
```
Comment (Plootonix) [inline GrizzFit/src/screens/Main.js:25]: thank you for deleting all this english

**Review by Plootonix — APPROVED** [review]
It's good

File: GrizzFit/src/components/WorkoutsListItem.js (hunk 1 above)
Comment (richardpluta) [reply GrizzFit/src/components/WorkoutsListItem.js:50]: understandably sus

File: GrizzFit/src/navigation/AppStack.js (hunk 3 above)
Comment (richardpluta) [reply GrizzFit/src/navigation/AppStack.js:33]: (;

File: GrizzFit/src/helpers/Helpers.js (hunk 2 above)
Comment (richardpluta) [reply GrizzFit/src/helpers/Helpers.js:7]: i agree

File: GrizzFit/src/providers/MainScreenProvider.js (hunk 4 above)
Comment (richardpluta) [reply GrizzFit/src/providers/MainScreenProvider.js:18]: ok master skywalker

File: GrizzFit/src/screens/Main.js (hunk 6 above)
Comment (richardpluta) [reply GrizzFit/src/screens/Main.js:28]: np

**Pull request merged**